src/screen/CreateAccount/MnemonicScreen.tsx

```diff
@@ -9,12 +9,18 @@ import FormLabel from 'presentational/FormLabel';
 import Padder from 'presentational/Padder';
 import {verifyMnemonicScreen} from 'src/navigation/routeKeys';
 import SubstrateSign from 'react-native-substrate-sign';
+import {createLogger} from 'src/utils';
+
+const logger = createLogger('Mnemonic');
 
 export function MnemonicScreen({navigation}: {navigation: NavigationProp<AccountsStackParamList>}) {
   const [mnemonic, setMnemonic] = React.useState<string>();
 
   React.useEffect(() => {
-    SubstrateSign.randomPhrase(12).then(setMnemonic);
+    SubstrateSign.randomPhrase(12).then((mnemonic) => {
+      logger.debug('Mnemonic seed ::: ', mnemonic);
```

Comment:
let's remove this or only log in development

Reply:
This is to see the mnemonic so that we can verify while testing without writing it down somewhere or taking a screenshot.

Reply:
Right now it's fine, but later we can integrate the logger with Sentry for example.. we just need to change the transport in the config and we may forget that we are logging the mnemonic. Debug is just the priority level so you can filter in this kind of tools